proposals/simd0221

```diff
@@ -0,0 +1,132 @@
+Below 33%, we potentially risk network safety because a supermajority of all SOL has explicitly not voted on any given block and this opens the edge case possibility of long range attacks.
+
+It also proposes the following bounds for the issuance rate:
+Upper Bound: The current Solana issuance curve (decreasing at a rate of 15% per year and will stop decreasing once nominal inflation is 1.5%).
+Lower Bound: 0%
+
+Increases or decreases in inflation should be proportional to the magnitude of the difference between the actual staking participation rate and the target rate (for example, 50% as per this proposal).
+
+This approach would allow for a more dynamic response to fluctuations in staking participation. By aligning inflation adjustments with the actual deviation, network issuance better reflects the network’s real-time economic and security conditions.
+
+Inflation adjustment function:
+
+Δi= k * Δs
+
+Δi = Inflation change for the new epoch
+k = Speed Co-efficient
+Δs = Staking Participation Rate (s) at the start of epoch – Target Staking Participation Rate (T)
+
+inew = max (0%, min (current issuance curve, ilast + Δi)
+
+ilast = Inflation in the last epoch
+inew = Inflation in the new epoch
+current issuance curve = inflation defined by current Solana issuance curve
+
+This proposal sets k = 0.05 per annum. So, for each extra percentage point higher/lower in staking participation rate, inflation would come down/go up by 0.05% p.a. in the next epoch. With the current staking participation rate of ~70%, the network would see a reduction of inflation of 1% p.a. in the next epoch. On the other hand, with a hypothetical staking participation rate of say 40%, the network would see an increase of inflation of 0.5% p.a. in the next epoch.
+
+The max function ensures that inflation is at least zero, and the min function ensures that the inflation does not rise above the current issuance curve.
+This design offers several key benefits:
+Consensus Safety: Adjusting inflation based on staking participation ensures sufficient validator incentives to maintain network security, prioritizing consensus safety.
+Market-Based Flexibility: The model adapts to the network's economic activity, making it more responsive to changing market conditions. It’s possible to imagine a future where stakers are earning enough from MEV that no SOL emissions are necessary. 
+Validator Retention: It accommodates Solana-aligned validators who are willing to stake even with lower emissions, recognizing that they can earn more through MEV in higher economic activity ecosystems.
+This dynamic approach balances the need for a secure, decentralized network with the flexibility to thrive in a competitive market.
+##Alternatives Considered
```

Comment:
was an alternative considered based on burn rate instead of stake rate? throw out [simd0096](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0096-reward-collected-priority-fee-in-entirety.md), add something [simd0109](https://github.com/solana-foundation/solana-improvement-documents/pull/109)-ish with burn. use utility as the measure, with security as the goal.

Reply:
> throw out [simd0096]

ser... we're like 4 days out

Reply:
it was always retarded